backend/src/main/java/quarano/department/activation/ActivationCodeException.java

```diff
@@ -1,42 +1,42 @@
 package quarano.department.activation;
 
+import lombok.Getter;
 import quarano.department.activation.ActivationCode.ActivationCodeIdentifier;
 
 /**
  * @author Oliver Drotbohm
  */
+@Getter
 public class ActivationCodeException extends RuntimeException {
 
-	private static final long serialVersionUID = -1862988574924745449L;
+		private static final long serialVersionUID = -1862988574924745449L;
```

Comment:
Looks weird. Please check my comment on the pull request. I am not sure if my code formatter is set as expected.